docs/readthedocs/source/doc/Nano/Overview/pytorch_inference.md

```diff
@@ -326,15 +346,22 @@ multi_model = InferenceOptimizer.to_multi_instance(model, num_processes=4, cores
 multi_model = InferenceOptimizer.to_multi_instance(model, cpu_for_each_process=[[0], [1], [2,3], [4,5]])
 ```
 
+```eval_rst
+.. note::
+    During multi-instance infernece, the context manager ``InferenceOptimizer.get_context(model=...)`` is not needed to be maunally added.
+```
+
 ## Automatic Context Management
-BigDL-Nano provides ``InferenceOptimizer.get_context(model=...)`` API to enable automatic context management for PyTorch inference. With only one line of code change, BigDL-Nano will automatically provide suitable context management for each accelerated model, it usually contains part of or all of following three types of context managers:
+BigDL-Nano provides ``InferenceOptimizer.get_context(model=...)`` API to enable automatic context management for PyTorch inference. With only one line of code change, BigDL-Nano will automatically provide suitable context management for each accelerated model optimized by ``InferenceOptimizer.trace``/``quantize``/``optimize``, it usually contains part of or all of following four types of context managers:
+
+1. ``torch.inference_mode(True)`` (when `torch` > 1.12) or ``torch.no_grad()`` (when `torch` <= 1.12) to disable gradients, which will be used for all models
```

Comment:
There are still small problem here.
For PT<=1.12, only **auto mixed precision inferencing** will use torch.no_grad()
That it, for non-bf16 model, we will always use `torch.inference_mode(True)`.

Reply:
Has revised. Thank you!